data/Abhinav-6.json

```
@@ -0,0 +1,32 @@
+{
+  "name": "Abhinav Kumar",
+  "type": "personal",
+  "displayStatsPublic": true,
+  "bio": "Frontend developer | diving deep into **Node.js** | **TypeScript** Enthusiast",
+  
+  "tags": [
+    "Open Source",
+    "Javascript",
+    "Typescript",
+    "NextJS",
+    "C++",
+    "Node.js"
+  ],
+  "socials": [
+    { "icon": "FaTwitter", "url": "https://twitter.com/_Abhinav6" },
+    { "icon": "FaGithub", "url": "https://github.com/Abhinav-6" },
```

Extra comma in here (if I'm not wrong).
```suggestion
    { "icon": "FaGithub", "url": "https://github.com/Abhinav-6" }
```